publish.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
```

Comment:
i know it makes it a bit longer but we should pull here first, to avoid using an old publish image.

`docker pull registry.tld/conjurinc/publish-rubygem`

Reply:
good catch!